### ISSUE TITLE
re-rangking

### ISSUE THREAD

**Issue by CompliceU** [layumi/Person_reID_baseline_pytorch#266]
Hello, I want to use your re-ranking to rearrange the search results, but the amount of data is huge, I don’t know how to deal with it. @layumi

**Comment by layumi** [comment]
@CompliceU 
Yes. re-ranking needs to load the similarity metric into the memory. 
You may try float16 or low-precision to save the memory.
